src/roles/common/templates/nuhs.j2

```diff
@@ -52,10 +54,10 @@ external_interface_networks:
 {% for interface in nuh_external_interfaces %}
 {% for external_if in item.external_interface_list %}
 {% if interface.name in external_if %}
-  - state: new
-    name: "{{ interface.external_bridge }}"
+  - network_name: "{{ interface.external_bridge }}"
     connected: true
```

Comment:
You can remove "connected: true" and "start_connected: true" here as they are set in the module that adds the extra interface adapters

Reply:
Done.

Reply:
Done!